Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/ManipulatorInteractionRequestBus.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzToolsFramework/Viewport/ViewportTypes.h>
+
+namespace EMStudio
+{
+    class ManipulatorRequests : public AZ::EBusTraits
```

Comment:
nit: I know this is namespaces, but it might be best to include EMFX in the name of the type to make searches less confusing. It feels like this would be easy to mistake for a more central type.

Also it would be good to chat about why this is needed, as I think we should be able to avoid creating this new bus and use an existing one (likely in `ViewportTypes.h` or `ViewportMessages.h` in `AzToolsFramework`

Reply:
1. EMStudio is an already exist namespace emfx uses for all it's editor code. We keep it under EMStudio to make the namespace shorter and distinguish from namespace EMotionFX (which is mostly for runtime stuff)
2.  The only thing close to what I want is the MouseViewportRequests which is not a bus but an interface. It's used in EditorComponentModeBus.h which seems to intertwine with logic with editor component, so i write a new bus for it. Would love to see a better idea.